tests/attr/test_feature_ablation.py

```diff
@@ -13,6 +16,7 @@
     BasicModel_MultiLayer_MultiInput,
 )
 from .helpers.utils import assertTensorAlmostEqual, BaseTest
+from captum.attr._utils.typing import TensorOrTupleOfTensors
 
 
 class Test(BaseTest):
```

Comment:
Can you add type hints to all the test_.. methods? Most of these will just have a None return type.

Reply:
Hi Vivek, I added type hint to all but one method on line 171, test_simple_multi_input_conv(self). If adding -> None to it, four errors appears as below. The errors on 174 and 179 seems related to the functions (torch.ones_like, and torch.type) that that are not defined in these two files, so I left this method blank. BTW, the errors on 176 and 183 goes contradict to the definition of inputs in the feature_ablation, which is supposed to be tensor or tuple of tensors. Thus, I left this method blank. 

Erros: 
tests/attr/test_feature_ablation.py:174: error: Argument 1 to "type" of "Tensor" has incompatible type "Type[FloatTensor]"; expected "Union[str, dtype, None]"
tests/attr/test_feature_ablation.py:176: error: Value of type variable "TensorOrTupleOfTensors" of "_ablation_test_assert" of "Test" cannot be "Tuple[Union[str, Tensor], Tensor]"
tests/attr/test_feature_ablation.py:179: error: Argument 1 to "ones_like" has incompatible type "Union[str, Tensor]"; expected "Tensor"
tests/attr/test_feature_ablation.py:183: error: Value of type variable "TensorOrTupleOfTensors" of "_ablation_test_assert" of "Test" cannot be "Tuple[Union[str, Tensor], Tensor]"

Reply:
You can add the type hint to that method, the only change you need to make for these errors to be resolved is replace line 173, with:
`inp = torch.arange(16, dtype=torch.float).view(1, 1, 4, 4)`

Reply:
revised and no more errors!